docs/using-gitbase/configuration.md

```diff
@@ -41,14 +41,18 @@ Help Options:
   -h, --help          Show this help message
 
 [server command options]
-      -v              Activates the verbose mode
-      -g, --git=      Path where the git repositories are located, multiple directories can be defined. Accepts globs.
-          --siva=     Path where the siva repositories are located, multiple directories can be defined. Accepts globs.
-      -h, --host=     Host where the server is going to listen (default: localhost)
-      -p, --port=     Port where the server is going to listen (default: 3306)
-      -u, --user=     User name used for connection (default: root)
-      -P, --password= Password used for connection
-          --pilosa=   URL to your pilosa server (default: http://localhost:10101)
-      -i, --index=    Directory where the gitbase indexes information will be persisted. (default: /var/lib/gitbase/index)
-          --no-squash Disables the table squashing.
+      -v                    Activates the verbose mode
```

Comment:
At the top of this file, we have a table with several env variables. Maybe we should add the new ones there too.